compiler_opt/es/gradient_ascent_optimization_algorithms.py

```diff
@@ -40,7 +42,8 @@ class GAOptimizer(metaclass=abc.ABCMeta):
   """
 
   @abc.abstractmethod
-  def run_step(self, current_input, gradient):
+  def run_step(self, current_input: npt.NDArray,
+               gradient: npt.NDArray[np.float32]) -> npt.NDArray:
```

Comment:
i guess you should update other places where you have np.NDArray to np.NDArray[np.float32]?

Reply:
They have been updated now